templates/pkg/crd_sdk.go.tpl

```diff
@@ -228,6 +238,7 @@ func (rm *resourceManager) sdkUpdate(
 {{ end }}
 
 {{ $setCode := GoCodeSetUpdateOutput .CRD "resp" "ko.Status" 1 }}
+{{ $setCustomOutputOperation := .CRD.GetCustomSetOutputOperation .CRD.Ops.Update }}
```

Comment:
ditto.

Reply:
Moved the code and used `{{ if $setOutputCustomMethodName := ` to combine into one statement.